clusterloader2/pkg/framework/framework.go

```diff
@@ -33,7 +33,7 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 )
 
-const (
+var (
```

Comment:
This shouldn't be global variable.

It should be a field of framework.Framework.

Reply:
Done.